lib/backlogs_hooks.rb

```diff
@@ -16,10 +16,10 @@ def exception(context, ex)
       def helper_issues_show_detail_after_setting(context={ })
       	begin
           if context[:detail].prop_key == 'release_id'
-            r = RbRelease.find_by_id(context[:detail].value)
+            r = RbRelease.find_by_id(context[:detail].value) unless context[:detail].value.include?('.')
```

Comment:
Not all release names contain dots, so a couple of alternative ideas - is `context[:detail].value` an integer rather than string on the first call? If so then perhaps do a type check, else perhaps change to `if context[:detail].value =~ /\A\d+\z/` instead.